client/src/components/pages/ProfileUpdate.tsx

```diff
@@ -228,6 +230,24 @@ const Screen: FC = () => {
     );
   };
 
+  const updateProfileToast = (): void => {
+    snackbar?.show({
+      text: getString('UPDATE_PROFILE'),
+      containerStyle: {
+        backgroundColor: `${theme.toastBackground}`,
```

Comment:
I think we can safely remove template string syntax.
```suggestion
        backgroundColor: theme.toastBackground,
```

Reply:
I will change this code at the next commit!
Thank you  👍